### PR TITLE
feat:Fees Register API keyword change to GeneralAppWithoutNotice

### PR DESCRIPTION
**Pull request by nkoranne**
**Before creating a pull request make sure that:**

- [ ] PR's title includes JIRA ticket number and follow [Conventional Commits](https://www.conventionalcommits.org/) specification
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by joez-f** [comment]
there are some changes to fpla-docker in this PR, are they intentional?

**Comment by nkoranne** [comment]
There is a wire mock change in c2-without-notice.json as well, not sure why that file not showing in this PR :(

**Comment by tomaszpowroznik** [comment]
> There is a wire mock change in c2-without-notice.json as well, not sure why that file not showing in this PR :(
this change is not correct, it reverts some of changes done in fpl docker recently

**Comment by tomaszpowroznik** [comment]
no changes in fpla-docker is needed for this.

File: service/src/main/resources/application.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ fees-register:
       event: general application
       jurisdiction1: family
       jurisdiction2: family court
-      keyword: without-notice
+      keyword: ${FEES_REGISTER_C2_WITHOUT_NOTICE_KEYWORD:GeneralAppWithoutNotice}
```
Comment (lewisbirks) [inline service/src/main/resources/application.yaml:35]: Don't think that the whole `${FEES_REGISTER_C2_WITHOUT_NOTICE_KEYWORD:GeneralAppWithoutNotice}` is needed, just `GeneralAppWithoutNotice`.
From my understanding as the env variable should map directly to the property spring should recognise this and give the env variable priority over the `application.yaml` value.

Comment (lewisbirks) [reply service/src/main/resources/application.yaml:35]: Ahh it would need to be called `FEES_REGISTER_PARAMETERS_C2_WITHOUT_NOTICE_KEYWORD`

Comment (tomaszpowroznik) [reply service/src/main/resources/application.yaml:35]: @lewisbirks are you sure about this? Code does not directly goes this this property

Comment (lewisbirks) [reply service/src/main/resources/application.yaml:35]: Chatted with tomasz and we can leave as is, just more work for a temp thing than is needed.

Comment (tomaszpowroznik) [reply service/src/main/resources/application.yaml:35]: i would prefer to keep explicit env variable, if somebody decide to refactor name in application xml then  env variable wont match and prod config can be broken without even noticing it. 
Thats nice feature, but i guess quite risky if you dont now context

Comment (akomisarek) [reply service/src/main/resources/application.yaml:35]: @nkoranne  As a safer approach we could leave old default and ovewrite with the new value in flux demo. 

But that would work if the overwriting is confirmed and we have got flux ready for prod to use the old value. (which can be merged beforehand)

**Pull request merged**